chunk/chunk_store_test.go

```diff
@@ -75,7 +79,8 @@ func TestChunkStore(t *testing.T) {
 			"bar":  "beep",
 			"toms": "code",
 		},
-		Data: []byte{},
+		Encoding: chunk.DoubleDelta,
+		Data:     chunks[0],
```

Comment:
Added test.  Not going to exercise metadataInIndex, as I'm going to remove that code at some point.